deploy/OSD-6246-flowcontrol-fix/config.yaml

```diff
@@ -3,7 +3,7 @@ selectorSyncSet:
   matchExpressions:
   - key: hive.openshift.io/version-major-minor
     operator: In
-    values: ["4.5"]
+    values: ["4.5","4.6"]
   - key: hive.openshift.io/version-major-minor-patch
     operator: NotIn
-    values: ["4.5.99"] # this should be replaced with the fixed version
+    values: ["4.6.13"]
```

Comment:
Need to enumerate all the versions that have the fix.  For example, with 4.6.14+ this SSS will match.

Reply:
Added 4.6.15 and 4.6.16 based on https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/ for now.